stdlib/printexc.mli

```diff
@@ -277,6 +285,12 @@ val convert_raw_backtrace_slot : raw_backtrace_slot -> backtrace_slot
 *)
 
 
+val get_raw_backtrace_next_slot : raw_backtrace -> raw_backtrace_slot option
+(** [get_raw_backtrace_next_slot slot] returns the next slot inlined, if any.
+
```

Comment:
Can you improve this comment?  I presume you mean that this function traverses the inlining stack at the given slot?